packages/algoliasearch-helper/src/requestBuilder.js

```diff
@@ -123,6 +123,24 @@ var requestBuilder = {
     return queries;
   },
 
+  /**
+   * Get all the queries to send to the client, those queries can used directly
+   * with the Algolia client.
+   * @private
+   * @param  {SearchParameters} state The state from which to get the queries
+   * @return {object[]} The queries
+   */
+  _getCompositionQueries: function getQueries(state) {
```

Comment:
Should this be tested for the combination of outcomes happening in `_getCompositionHitsSearchParams`? This could also be done in a followup PR.